op-supernode/supernode/activity/superroot/superroot.go

```diff
@@ -44,37 +44,42 @@ type L2WithRequiredL1 struct {
 	MinRequiredL1 eth.BlockID
 }
 
-// atTimestampResponse is the response superroot_atTimestamp
+// AtTimestampResponse is the response superroot_atTimestamp
 // it contains:
 // - CurrentL1Derived: the current L1 block that each chain has derived up to (without any verification)
 // - CurrentL1Verified: the current L1 block that each verifier has processed up to
 // - VerifiedAtTimestamp: the L2 blocks which are fully verified at the given timestamp, and the minimum L1 block at which verification is possible
 // - OptimisticAtTimestamp: the L2 blocks which would be applied if verification were assumed to be successful, and their L1 sources
 // - SuperRoot: the superroot at the given timestamp using verified L2 blocks
-type atTimestampResponse struct {
+type AtTimestampResponse struct {
+	// TODO: We should probably specify json names for these
 	CurrentL1Derived      map[eth.ChainID]eth.BlockID
 	CurrentL1Verified     map[string]eth.BlockID
 	VerifiedAtTimestamp   map[eth.ChainID]L2WithRequiredL1
 	OptimisticAtTimestamp map[eth.ChainID]OutputWithSource
 	MinCurrentL1          eth.BlockID
-	MinVerifiedRequiredL1 eth.BlockID
-	SuperRoot             eth.Bytes32
+	MaxVerifiedRequiredL1 eth.BlockID
+	// Not entirely sure it's a good idea to include the actual super implementation here rather than extracting the data from it
+	// But it sure is convenient.
+	// If we do this we need to sepecify json names in the eth.SuperV1 struct
+	Super     eth.Super
+	SuperRoot eth.Bytes32
 }
 
 // AtTimestamp computes the super-root at the given timestamp, plus additional information about the current L1s, verified L2s, and optimistic L2s
-func (api *superrootAPI) AtTimestamp(ctx context.Context, timestamp uint64) (atTimestampResponse, error) {
+func (api *superrootAPI) AtTimestamp(ctx context.Context, timestamp uint64) (AtTimestampResponse, error) {
```

Comment:
I suspect to get the expected handling of numbers we should be using `timestamp hexutil.Uint64` here rather than just `uint64`. JSON_RPC (sadly) uses hex numbers and it would be very confusing if we wound up with some APIs needing hex and some decimal.